src/backend/call-service-in-docker.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+#set -x
+
+#FSDIR="/opt/obs/SourceServiceSystem"
+DOCKER_IMAGE=src-service-docker
+SERVICES_DIR="/srv/obs/service/"
+
+WITH_NET="0"
+COMMAND="$1"
+shift
+case "$COMMAND" in
+  */download_url|*/download_src_package|*/update_source|*/download_files|*/generator_pom)
+    WITH_NET="1"
+    ;;
+  */tar_scm|*/obs_scm)
+    SCM_COMMAND=1
+    WITH_NET="1"
+  ;;
+esac
+
+while [ $# -gt 0 ]; do
+  case $1 in 
+    --scm)
+      PARAM_SCM=$2
+    ;;
+    --scm=*)
+      PARAM_SCM=$1
+      PARAM_SCM=${PARAM_SCM#--scm=}
+    ;;
+    --url)
+      PARAM_URL=$2
+    ;;
+    --url=*)
+      PARAM_URL=$1
+      PARAM_URL=${PARAM_URL#--url=}
+    ;;
+  esac
+  if [ "$1" == "--outdir" ] ; then
+     shift
+     OUTDIR="$1"
+  else
+     COMMAND="$COMMAND '${1//\'/_}'"
+  fi
+  shift
+done
+
+if [ -z "$OUTDIR" ] ; then
+  echo "ERROR: no outdir given"
+  exit 1
+fi
+
+MOUNTDIR=`dirname $OUTDIR`
+RETURN="0"
+[ -d $MOUNTDIR ] || mkdir -p $MOUNTDIR
+# set -x
+
+INNEROUTDIR="/tmp/$$/out"
+OUTEROUTDIR="$MOUNTDIR/out"
+INNERSRCDIR="/tmp/$$/src"
+OUTERSRCDIR="$MOUNTDIR/src"
+INNERSCRIPTDIR="/tmp/$$/scripts"
+INNERSCRIPT="$INNERSCRIPTDIR/inner.sh"
+INNERGITCACHE="/tmp/git-cache"
```

Comment:
We should not use /tmp/ since it is world writable. Better use something like /var/cache/obs/service.docker/$$  or alike